internal/handlers/pg/msg_findandmodify.go

```diff
@@ -381,7 +382,16 @@ func prepareFindAndModifyParams(document *types.Document) (*findAndModifyParams,
 
 	var hasUpdateOperators bool
 	for k := range update.Map() {
-		if _, ok := common.UpdateOperators[k]; ok {
+		if slices.Contains(
+			[]string{
+				"$currentDate",
+				"$set",
+				"$setOnInsert",
+				"$unset",
+				"$inc",
+			},
+			k,
+		) {
```

Comment:
@w84thesun That's the third place, in addition to `UpdateOperators` and `checkAllModifiersSupported`. I'm pretty sure we don't need it :)